docs/installation/installation_google_cloud_kubernetes.md

```diff
@@ -0,0 +1,86 @@
+# Deploying loklak on Google Cloud Platform with Kubernetes
+> **Kubernetes** is an open-source system for automating deployment, scaling, and management of containerized applications.
+
+Source: https://kubernetes.io/
+
+Follow these 10 steps to get an instance of loklak running on Google Cloud platform using Kubernetes.
+
+**Note**: The document uses predefined names for various properties like project ID and docker image name to avoid confusion. You may want to change them when needed.
+
+### 1. Create Google Cloud Platform Account
+Visit https://cloud.google.com/free/ and follow on-screen instructions to create an account and get the free trial worth $300.
+
+### 2. Create a Project
+Create a new project with desired name and ID.
+
+![kub1](https://cloud.githubusercontent.com/assets/10860278/26034852/c7ff5cac-38e0-11e7-92fb-1fa711a4c05c.png)
+
+### 3. Create a Container Engine for the Project
+Go to https://console.cloud.google.com/kubernetes/list and wait for the container engine to get ready. You may need to select the project from the project section in the top-left corner.
+
+### 4. Open the Google Cloud Shell
+In the top-right panel, click the shell button.
+
+![screenshot from 2017-05-14 20-11-27](https://cloud.githubusercontent.com/assets/10860278/26034895/9703f85a-38e1-11e7-8dc1-81e38a5205d0.png)
+
+You'll be greeted by an online console.
+
+![screenshot from 2017-05-14 20-19-07](https://cloud.githubusercontent.com/assets/10860278/26034960/a8147dbc-38e2-11e7-87a1-b9fef076ee6f.png)
+
+### 5. Clone the loklak project
+```sh
+$ git clone https://github.com/loklak/loklak_server.git
+$ cd loklak_server/
+```
+
+### 6. Build the Docker Image and Push to Google Container Registry
+```sh
+$ docker build -t gcr.io/singhpratyush-loklak/loklak:v1 .
+$ gcloud docker -- push gcr.io/singhpratyush-loklak/loklak:v1
```

Comment:
@kavithaenair: I have left it that way intentionally to avoid any confusion for people referring it.